runtime/frequency/src/lib.rs

```diff
@@ -947,17 +948,25 @@ impl pallet_passkey::Config for Runtime {
 	type Currency = Balances;
 }
 
-#[cfg(any(not(feature = "frequency-no-relay"), feature = "frequency-lint-check"))]
+#[cfg(any(
+	feature = "frequency",
+	feature = "runtime-benchmarks",
+	feature = "frequency-lint-check",
+))]
 /// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
 /// into the relay chain.
 const UNINCLUDED_SEGMENT_CAPACITY: u32 = 1;
```

Comment:
Does this mean we are technically not supporting async backing in mainnet?

Reply:
Thats correct, at the moment we are not supporting async backing in mainnet.